pincer/utils/conversion.py

```diff
@@ -29,22 +29,24 @@ def construct_client_dict(client: Client, data: Dict) -> Dict:
     return {**data, "_client": client}
 
 
-def remove_none(obj: Union[List, Dict, Set]) -> Union[List, Dict, Set]:
+def remove_none(obj: Union[List, Dict, Set, Tuple]) -> Union[List, Dict, Set, Tuple]:
```

Comment:
`List[Any]`, `Dict[Any]`, `Tuple[Any]`

Reply:
iirc, Generic Types are implied to have `Any` when not used "genericly"

Reply:
^ its more readable for sure